2023-03-16 20:12:14,615 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Configure stats pid to 59759
2023-03-16 20:12:14,615 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/.config/wandb/settings
2023-03-16 20:12:14,615 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/settings
2023-03-16 20:12:14,615 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-16 20:12:14,615 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-16 20:12:14,615 INFO    MainThread:59759 [wandb_init.py:_log_setup():492] Logging user logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230316_201214-lda2o0v5/logs/debug.log
2023-03-16 20:12:14,615 INFO    MainThread:59759 [wandb_init.py:_log_setup():493] Logging internal logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230316_201214-lda2o0v5/logs/debug-internal.log
2023-03-16 20:12:14,616 INFO    MainThread:59759 [wandb_init.py:init():532] calling init triggers
2023-03-16 20:12:14,616 INFO    MainThread:59759 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-16 20:12:14,616 INFO    MainThread:59759 [wandb_init.py:init():554] re-initializing run, found existing run on stack: tqg6xlr0
2023-03-16 20:12:14,617 INFO    MainThread:59759 [wandb_run.py:_finish():1854] finishing run agnostic/rest_run_pong/tqg6xlr0
2023-03-16 20:12:14,617 ERROR   MainThread:59759 [jupyter.py:save_history():441] Run pip install nbformat to save notebook history
2023-03-16 20:12:14,617 INFO    MainThread:59759 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-16 20:12:14,617 INFO    MainThread:59759 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-16 20:12:14,617 INFO    MainThread:59759 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-16 20:12:14,618 INFO    MainThread:59759 [wandb_run.py:_restore():2066] restore
2023-03-16 20:12:14,618 INFO    MainThread:59759 [wandb_run.py:_restore():2072] restore done
2023-03-16 20:12:21,110 INFO    MainThread:59759 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-16 20:12:21,111 INFO    MainThread:59759 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-16 20:12:21,120 INFO    MainThread:59759 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-16 20:12:21,133 INFO    MainThread:59759 [wandb_init.py:init():588] starting backend
2023-03-16 20:12:21,134 INFO    MainThread:59759 [wandb_init.py:init():592] setting up manager
2023-03-16 20:12:21,146 INFO    MainThread:59759 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-16 20:12:21,152 INFO    MainThread:59759 [wandb_init.py:init():599] backend started and connected
2023-03-16 20:12:21,158 INFO    MainThread:59759 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-16 20:12:21,158 INFO    MainThread:59759 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-16 20:12:21,158 INFO    MainThread:59759 [wandb_init.py:init():687] updated telemetry
2023-03-16 20:12:21,237 INFO    MainThread:59759 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-16 20:12:22,532 INFO    MainThread:59759 [wandb_run.py:_on_init():2134] communicating current version
2023-03-16 20:12:22,614 INFO    MainThread:59759 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-16 20:12:22,614 INFO    MainThread:59759 [wandb_init.py:init():775] starting run threads in backend
2023-03-16 20:12:22,713 INFO    MainThread:59759 [wandb_run.py:_console_start():2114] atexit reg
2023-03-16 20:12:22,713 INFO    MainThread:59759 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-16 20:12:22,713 INFO    MainThread:59759 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-16 20:12:22,714 INFO    MainThread:59759 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-16 20:12:22,715 INFO    MainThread:59759 [wandb_init.py:init():817] run started, returning control to user process
2023-03-16 20:13:49,744 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Configure stats pid to 59759
2023-03-16 20:13:49,744 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/.config/wandb/settings
2023-03-16 20:13:49,744 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/settings
2023-03-16 20:13:49,744 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-16 20:13:49,744 INFO    MainThread:59759 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-16 20:13:49,744 INFO    MainThread:59759 [wandb_init.py:_log_setup():492] Logging user logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230316_201349-c7txoth6/logs/debug.log
2023-03-16 20:13:49,745 INFO    MainThread:59759 [wandb_init.py:_log_setup():493] Logging internal logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230316_201349-c7txoth6/logs/debug-internal.log
2023-03-16 20:13:49,745 INFO    MainThread:59759 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fb7ec20d8e0>
2023-03-16 20:13:49,745 INFO    MainThread:59759 [wandb_init.py:init():532] calling init triggers
2023-03-16 20:13:49,745 INFO    MainThread:59759 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-16 20:13:49,745 INFO    MainThread:59759 [wandb_init.py:init():554] re-initializing run, found existing run on stack: lda2o0v5
2023-03-16 20:13:49,747 INFO    MainThread:59759 [wandb_run.py:_finish():1854] finishing run agnostic/rest_run_pong/lda2o0v5
2023-03-16 20:13:49,747 INFO    MainThread:59759 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-16 20:13:49,748 INFO    MainThread:59759 [wandb_run.py:_restore():2066] restore
2023-03-16 20:13:49,748 INFO    MainThread:59759 [wandb_run.py:_restore():2072] restore done
2023-03-16 20:13:56,220 INFO    MainThread:59759 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-16 20:13:56,221 INFO    MainThread:59759 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-16 20:13:56,231 INFO    MainThread:59759 [wandb_run.py:_footer_sync_info():3398] logging synced files
